packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -402,6 +417,34 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
         return app
     }
 
+    async _uploadServiceLogs() {
+        const fileStream = fs.createReadStream(path.join(process.cwd(), LOGS_FILE))
```

Comment:
For `path.join(process.cwd(), LOGS_FILE)`, can we use the attribute `logFilePath` from BstackLogger?